src/handlers/next/saleor-async-webhook.test.ts

```diff
@@ -0,0 +1,75 @@
+import { createMocks } from "node-mocks-http";
+import { describe, expect, it, vi } from "vitest";
+
+import { APL } from "../../APL";
+import { NextWebhookApiHandler, SaleorAsyncWebhook } from "./saleor-async-webhook";
+
+const webhookPath = "api/webhooks/product-updated";
```

Comment:
what happens if I add `/` at the beginning?

Reply:
Both will result in the same URL